nautobot/dcim/templates/dcim/inventoryitem_add.html

```diff
@@ -0,0 +1,65 @@
+{% extends 'base.html' %}
```

Comment:
Is that the best template for this? I would have thought we'd be able to use generic/object_create.html?